### DISCUSSION TITLE
How to change the default ` environment variables` ?

### DISCUSSION THREAD

**Discussion by premdasvm** [nektos/act#1953] [Q&A]
How can i change the default value of `${{ github.actor }}` from `nektos/act` to something else?

**Comment by shubhbapna** [comment]
You should be able to change it by overriding the corresponding env variable like this `--env GITHUB_ACTOR=actor`. Although maybe for `github.actor` this was missed and it might not work:
https://github.com/nektos/act/blob/master/pkg/runner/run_context.go#L624

**Reply by premdasvm** [reply]
It doesn't work.
